python/ray/autoscaler/v2/instance_manager/config.py

```diff
@@ -152,25 +217,33 @@ def get_provider_instance_type(self, instance_type_name: str) -> str:
         else:
             raise ValueError(f"Unknown provider {provider}")
 
-    @property
-    def restart_only(self) -> bool:
-        return self._node_configs.get("restart_only", False)
-
-    @property
-    def no_restart(self) -> bool:
-        return self._node_configs.get("no_restart", False)
+    def get_cluster_config(self) -> Optional[ClusterConfig]:
```

Comment:
What's the difference between AutoscalingCofig and ClusterConfig?

Reply:
I always had the mapping `cluster.yaml <-> ClusterConfig` in my head, hence the confusion.